sale_import_rest/components/sale_import_service.py

```diff
@@ -0,0 +1,36 @@
+#  Copyright (c) Akretion 2020
+#  License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+from odoo import _
+from odoo.exceptions import ValidationError
+
+from odoo.addons.component.core import Component
+
+
+class SaleImportService(Component):
+    _inherit = "base.rest.service"
+    _name = "sale.import.service"
+    _usage = "import"
+    _collection_name = "sale.import.endpoints"
+    _default_auth = "public"
```

Comment:
this should not be public you should use the auth api method (you can check example in shopinvader)